data-client/src/test/java/org/wikipedia/dataclient/mwapi/page/MwPageClientTest.java

```diff
@@ -19,6 +20,7 @@ public class MwPageClientTest extends BasePageClientTest {
         subject = new MwPageClient();
     }
 
+    @Ignore("MobileView API is deprecated. Remove test cases after fix")
```

Comment:
Please add a link to https://github.com/commons-app/apps-android-commons/issues/5165 here and at the similar lines, thanks!